litmus-portal/frontend/cypress/component/buttons.spec.tsx

```diff
@@ -35,12 +34,12 @@ describe('Button Outline', () => {
   });
   it('The button is enabled', () => {
     mount(
-      <ButtonOutline
-        isDisabled={false}
-        handleClick={() => console.log('Handle Click')}
+      <ButtonOutlined
+        disabled={false}
```

Comment:
Do we even need this `disabled={false}` line! Since it's enabled by default. Can we try running the test without this line and check once!